src/__tests__/ParseObject-test.js

```diff
@@ -908,6 +907,16 @@ describe('ParseObject', () => {
     });
   });
 
+  it('validates attributes on save()', async () => {
+    const o = new ParseObject('Listing');
+    try {
+      await o.save({ '$$$': 'o_O' });
+      expect(true).toBe(false);
```

Comment:
Looks odd, should it be asyncAwait?

Reply:
I wrote it this way since there is only an error code and no error message. I'll change it